symbulate/distributions.py

```diff
@@ -375,12 +394,21 @@ class Exponential(Distribution):
 
     def __init__(self, rate=1.0, scale=None):
         
-        if scale > 0:
-            self.scale = scale
+        if (rate is None) and (scale is None):
+            raise Exception("Input rate argument")
```

Comment:
rate will never be None because its default value is 1.0. The only one that is in danger of being None is scale.